vtkext/private/module/vtkF3DEXRReader.cxx

```diff
@@ -112,12 +175,8 @@ void vtkF3DEXRReader::ExecuteDataWithInformation(vtkDataObject* output, vtkInfor
   scalars->SetName("Pixels");
   float* dataPtr = scalars->GetPointer(0);
 
-  try
+  auto execute = [&](Imf::RgbaInputFile& file)
```

Comment:
maybe name that lambda something else to avoid confusion with the other execute lambda ?